CMakeLists.txt

```diff
@@ -86,6 +99,8 @@ ament_target_dependencies(${PROJECT_NAME}_exported
 target_link_libraries(${PROJECT_NAME}
   gRPC::grpc++_reflection
   gRPC::grpc++
+  ${OpenCV_LIBS}
+  /opt/intel/openvino_2023/runtime/lib/intel64/libopenvino.so
```

Comment:
The 2023 version of OpenVINO seems to already provide a package config as seen [here](https://github.com/openvinotoolkit/openvino/blob/releases/2023/3/cmake/templates/OpenVINOConfig.cmake.in). Can you double check if it is available in the system first before hardcoding the library path?